app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -43,6 +43,8 @@ def view_toolbar_filename
       'drift_view_tb'
     elsif %w(ems_container ems_infra).include?(@layout) && %w(main dashboard topology).include?(@display)
       'dashboard_summary_toggle_view_tb'
+    elsif %w(container_project).include?(@layout)
+      "#{@layout}_view_tb"
```

Comment:
> I don't get how the part entities have topology\dashboard views relates to role_allows??

I thought that if an entity (like ems) has a dashboard view available (maybe through `supports?`) then we should add the toolbar dashboard view button instead of picking specific entities with their toolbars.
or maybe we always have entities search for their `"#{@layout}_view_tb"` file.
Im just trying to prevent massive refactors later...